Google.PowerShell/Storage/GcsCmdlet.cs

```diff
@@ -24,12 +24,10 @@ public abstract class GcsCmdlet : GCloudCmdlet
         /// </summary>
         protected const string UTF8TextMimeType = "text/plain; charset=utf-8";
 
-        // TODO(chrsmith): Cache the storage service? Create it in OnProcessRecord every time? (So it does so once?)
-
-        protected StorageService GetStorageService()
-        {
-            return new StorageService(GetBaseClientServiceInitializer());
-        }
+        /// <summary>
+        /// The storage service.
```

Comment:
I'm not a huge fan of this, but don't have a good reason to oppose the change.

The initial design was so that we would explicitly control the lifetime of the service object. That would make it easier to mock, test, instrument, etc.

However, we could still probably do all of those same things by going through a property accessor. And this approach also cleans up a lot of the code.